### PR TITLE
Add a subcommand on the hil tool that report the version itself

### PR DESCRIPTION
**Pull request by xuhang57**
This patch relates to https://github.com/CCI-MOC/hil/issues/389
And this implementation is based on the first option on @zenhack 's list.

### CODE REVIEW

**Comment by xuhang57** [comment]
This would return: 
```
0.3.dev163+ga715d2e(.venv) ubuntu@lucas-hil:~/hil$ vim hil/cli.py
```

Any suggestion on what exactly the return value should look like? 

And on github tag:https://github.com/CCI-MOC/hil/releases
There is only a `0.2` release.

File: hil/cli.py [hunk 1]
```diff
@@ -37,7 +39,7 @@
 usage_dict = {}
 MIN_PORT_NUMBER = 1
 MAX_PORT_NUMBER = 2**16 - 1
-
+VERSION = pkg_resources.require('hil')[0].version
```
Comment (naved001) [inline hil/cli.py]: any reason why you are getting version here and not in the function itself?

Comment (xuhang57) [reply hil/cli.py]: I use Ian's code as an example:https://github.com/zenhack/simp_le/blob/0fc118de6c0fcad0e7a23a8e788a1e00995a1f5a/simp_le.py#L67

And I have seen people coding this way in other places. I guess it is a convention? But I understand we only use this once in the entire code base. Maybe using just the function makes sense?

Comment (naved001) [reply hil/cli.py]: Don't feel too strongly about this. If it's a convention which might enable some reuse in the future, I am okay with it.

Comment (xuhang57) [reply hil/cli.py]: hmm, @zenhack might know better here. I am not sure whether this is kinda a convention but it seems like so to me :)

Comment (zenhack) [reply hil/cli.py]: I don't have a strong opinion either. The code I linked to is in a project I inherited; I wasn't around for the decision on that one.

Re: output format, maybe just something like:

```
HIL version ${version_number}
```

You sound make sure to add a newline, so it doesn't end up on the same line as your prompt.

Comment (xuhang57) [reply hil/cli.py]: Sure, was planning to add that once I have an idea about how the output should look like. Should've mentioned that earlier. Thanks.

**Comment by xuhang57** [comment]
```
(.venv) ubuntu@lucas-hil:~/hil$ hil version
HIL version: 0.3.dev163+ga715d2e
```
Updated cli response formt. @zenhack @naved001

**Comment by naved001** [comment]
Any need to rebase since I think we modified cli.py in another PR which we merged (though in different places)?

**Comment by naved001** [comment]
There's a rebase and merge button too? Anybody used it before?

**Comment by zenhack** [comment]
Would prefer if you merged master back into your branch (don't care rebase vs. merge commit). I'd prefer not to do it on GitHub because that means we're pushing something to master that is not the exact same tree that travis is running against.

**Comment by zenhack** [comment]
...but I don't know that it's worth fussing about for something so small.

**Comment by naved001** [comment]
Alright, I'll merge.

**Pull request merged**